### ISSUE TITLE
Display the value instead of the key of the grid component in webform submissions

### ISSUE THREAD

**Issue by luxio** [signalpoint/webform#18]
It would be nice to display the value of the questions and options of a grid component when displaying the submission.

Maybe we could us a theming function for each component type for the submission display, similar to the Drupal webform module.

**Comment by signalpoint** [comment]
@luxio Yes, I agree on using a theme function specific for each component, and we could then fall back to the way it currently outputs it, if that particular component doesn't yet have a theme function.
